src/SourceBuild/tarball/content/tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/RemoveInternetSourcesFromNuGetConfig.cs

```diff
@@ -75,6 +76,9 @@ public override bool Execute()
             // Remove disabledPackageSources element so if any internal packages remain, they are used in source-build
             disabledPackageSourcesElement?.ReplaceNodes(new XElement("clear"));
 
+            // Do the same for packageSourceMappings, these are re-added in AddSourceToNuGetConfig
+            packageSourceMappingElement?.ReplaceNodes(new XElement("clear"));
```

Comment:
It doesn't seem right to remove these when building in online mode.  We should fall back to these sources if something isn't included in source-build.